edge-agent/docker/linux/amd64/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG base_tag=3.1.19-alpine3.14
+ARG base_tag=3.1-alpine
```

Comment:
I believe this new tag resolves to Alpine 3.15, which is a MINOR bump, not just PATCH. We probably want to use `3.1-alpine3.14` for now, and make an explicit move to 3.15 later, after some testing.

Reply:
Generally we bumped the MINOR version for alpine all the time. As long as the tests pass, we called it's good. From my experience the MINOR OS version bump for .NET Alpine is more frequent than other OS such as Ubuntu. I've seen when the alpine used 2 different MINOR versions within .NET Core releases.

@ancaantochi  Do you have any concerns?

Reply:
In another comment you said
> As far as this PR concerns, the 3.1-alpine tag is aliased to 3.1.22-alpine3.14

I see that now. Not sure where I got 3.15 from. Alpine 3.15 is available, but I don't see it any any dotnet/aspnet images. We can keep the base tag at `3.1-alpine`, and revisit if needed at any time.